client/components/Contact.js

```diff
@@ -63,6 +71,14 @@ class Contact extends Component {
     });
   }
 
+  handleCheck (event) {
+    let prevState = Object.assign({}, this.state.interest);
```

Comment:
Since this only contains the interest part of the state, rather than the whole state object, this should probably be called something like `const interest`

Reply:
Done